lib/Units.pm

```diff
@@ -230,11 +230,22 @@ our %known_units = (
 		factor => 0.0001,
 		m      => 3
 	},
+	# U.S./English volume units
 	cup => {
 		factor  => 0.000236588,
 		m       => 3,
 		aliases => ['cups']
 	},
+	pt => {
```

Comment:
Judging by the adjacent entries (`cup` and `gallon`), maybe we have a standard where the singular full word is the key, and other things are aliases. So `pint => ... and `quart => ... `. And the plural full word is the first listed alias.